dash_labs/plugins/pages.py

```diff
@@ -294,6 +297,40 @@ def _import_layouts_from_pages():
                 )
 
 
+def _register_page_from_markdown_file(filename, app):
+    """
+    Extracts and runs dash.register_page() from the "front matter" of a markdown file in the pages folder.
+    Front matter is yaml defined with three dashes:
+    ---
+    register_page:
+      path: "/home"
+      order: 2
+    MarkdownAIO:
+      exec: True
+      side_by_side: True
+    ---
+    """
+    import frontmatter
+    from dash_labs import MarkdownAIO
+
+    try:
+        md_page = frontmatter.load(filename)
+    except Exception as e:
+        print(f"Error reading {filename} \n{e}")
```

Comment:
we can remove this `try` / `except` - if there is an error, we should raise it

Reply:
This will probably change with the new parser....   But without the `try` / `except` it would raise the error, but the message did not include the filename

Reply:
good idea to include the filename in the error message. We'll still want the program to stop though, so in the future you can do things like raise a custom exception:
```
    try:
        md_page = frontmatter.load(filename)
    except Exception as e:
        raise Exception(f"Error parsing the YAML front matter of {filename}:\n{e}")
```